src/loaders.rs

```diff
@@ -213,6 +267,23 @@ mod tests {
 
     use quickcheck::quickcheck;
 
+    fn setup_django(py: Python<'_>) {
+        // Import the os module and set the DJANGO_SETTINGS_MODULE environment variable
+        let os_module = PyModule::import(py, "os").unwrap();
+        let environ = os_module.getattr("environ").unwrap();
+        environ
+            .call_method(
+                "setdefault",
+                ("DJANGO_SETTINGS_MODULE", "tests.settings"),
+                None,
+            )
+            .unwrap();
+
+        // Import the django module and call django.setup()
+        let django_module = PyModule::import(py, "django").unwrap();
+        django_module.call_method0("setup").unwrap();
+    }
```

Comment:
I think it would probably be simpler to use Python tests than try to configure Django from Rust. Is there a reason you preferred this approach?

Reply:
I used this approach to be able to test rust code following the old tests patterns. get_app_template_dirs calls get_app_configs from django which needs setup to be called from django first.
so in each test that use get_app_template_dirs I call setup_django to make sure that get_app_configs is called without any errors.